### ISSUE TITLE
multiple parts mesh

### ISSUE THREAD

**Issue by vinhphunguyen** [ahojukka5/AbaqusReader.jl#62]
It seems to me that AbaqusReader.jl cannot read an inp file containing multiple parts. Is that true?

**Comment by TeroFrondelius** [comment]
This is true. You can always export the traditional inp without assembly structure, thus there hasn't been a need for it earlier.

**Comment by ahojukka5** [comment]
Could we have here some minimal example of multiple part mesh? It's maybe not that big effort to implement, I think I have done it already one time. I really mean a minimal example, just showing the structure of the input file with couple of elements and nodes.

**Comment by vinhphunguyen** [comment]
Here is one example.

Best regards,
Phu

**Comment by ahojukka5** [comment]
GitHub is not accepting email attachments, please copy-paste the mesh in the actual mail.
